### DISCUSSION TITLE
pushgateway and custom headers

### DISCUSSION THREAD

**Discussion by ThorstenSuckow** [prometheus/pushgateway#586] [Q&A]
Is there any way to instruct pushgateway to send custom headers with each response?

TIA!

**Comment by beorn7** [comment]
Are you looking for responses when _pushing_ metrics to the /metrics/... endpoint, or for responses when _scraping_ the /metrics endpoint? (The former is PGW specific, but the latter is just a normal /metrics endpoint as exposed by any instrumented binary.)
